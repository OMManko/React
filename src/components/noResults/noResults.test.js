import { shallow } from 'enzyme';
import React from 'react';
import NoResults from './noResults';

describe('<NoResults />', () => {
    it('should render no results block and match shapshot', () => {
        const component = shallow(<NoResults />);
        expect(component).toMatchSnapshot();
    });
});
