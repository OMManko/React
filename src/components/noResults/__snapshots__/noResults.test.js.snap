// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoResults /> should render no results block and match shapshot 1`] = `ShallowWrapper {}`;
